File: python/51 Book.py
class Book:
    totalBooks=10
    def __init__(self, title, author, isbn):
        self.title=title
        self.author=author
        self.isbn=isbn
        Book.totalBooks+=1
    
    def borrowBook(self):
        if Book.totalBooks==0:
            print("Books unavailable")
        else:
            Book.totalBooks-=1
            print("Borrow successful")

    def returnBook(self):
        Book.totalBooks+=1
        print("Return successful")
    
    @classmethod
    def getTotalBooks(cls):
        return cls.totalBooks

b1=Book("1984", "George Orwell", 12345)
b2=Book("Animal Farm", "George Orwell", 67890)
b1.borrowBook()
print(Book.getTotalBooks())

File: java/51 Book.java
package code;
class Books{
	String title, author;
	int isbn;
	static int totalBooks=10;
	Books(String title,String author, int isbn){
		this.title=title;
		this.author=author;
		this.isbn=isbn;
		totalBooks++;
	}
	void borrowBook() {
		totalBooks--;
		System.out.println("Borrow successful");
	}
	void returnBook() {
		totalBooks++;
		System.out.println("Return successful");
	}
	static int getTotalBooks() {
		return totalBooks;
	}
}
public class Test {
	public static void main(String[] args) {
		Books b1=new Books("1984", "George Orwell", 12345);
		Books b2=new Books("Animal Farm", "George Orwell", 67890);
		b1.borrowBook();
		System.out.print(Books.getTotalBooks());
	}
}
